geopandas/tools/tests/test_sjoin.py

```diff
@@ -117,6 +117,18 @@ def test_left(self, op, dfs):
 
         assert_frame_equal(res, exp)
 
+        # Check empty joins on left
```

Comment:
Can you maybe make this just a separate test? (`test_empty_join` or something)

Reply:
sure, yes